### DISCUSSION TITLE
Add 1 to Int field

### DISCUSSION THREAD

**Discussion by Amgelo563** [prisma/prisma#13642] [Q&A] [answered]
Hello, sorry if this is included on the documentation but I couldn't find it. How should I go for adding 1 to an Int field (MySQL)? Google mentions this:
```sql
UPDATE x SET int = int + 1 WHERE id = 'ID'
```
But I'm not sure how should I go for "translating" that on the Prisma Client. Thanks for the help in advance!

**Comment by Amgelo563** [comment, marked as answer]
Nevermind, found it now on the [Client API Reference](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#atomic-number-operations), thanks anyways!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
